bin/stringsCompletenessTest.py

```diff
@@ -97,6 +142,7 @@ def check_strings_in_code(l10n_properties, not_found_string_list, **list):
     # directories to look in
     def_code_dirs = [
         "add-on/chrome",
+        "add-on/chrome/modules",
         "add-on/panels/js",
         "add-on/panels/vendor",
```

Comment:
Do we expect strings from `vendor` files? I would think not, so it could be removed. But if we keep this for `panels`, we probably want to include `shared`/`standalone` equivalent directories too?